tox.ini

```diff
@@ -0,0 +1,7 @@
+[tox]
+envlist = py27,py36,py37,py38,py39,py310,py311,py312,pypy3
```

Comment:
Was it intentional to add back in python 2.7 and 3.6 into the checks? I think the last release bumped support to >= 3.7.

Reply:
No. That's an oversight. I'll fix it right away.

Reply:
Done